api/src/pcapi/core/subscription/dms/api.py

```diff
@@ -400,21 +401,14 @@ def parse_and_handle_dms_application(
 
     user = find_user_by_email(user_email)
     if not user:
-        # TODO: Handle this error differently, when we accept applications from DMS before user creation
+        _process_user_not_found_error(user_email, application_id, procedure_id)
```

Comment:
Le problème venait du fait qu'on n'appelait pas la fonction qui permet de créer des OrphanDMSApplications?

Reply:
Yep
Elle est appelée dans `_process_user_not_found_error`

Reply:
Le problème n'était cependant que temporaire, le cron passe et recolle le dossier avec l'utilisateur s'il existe, ou crée un orphelein

Reply:
ça roule. ça me semble bien ! Je ne vois pas précisément dans quelle partie du code on rattache une nouvelle inscription in-app à une OrphanApplication, mais je suppose que ça faisait partie d'une autre PR de toute façon.